backend/src/services/bambora.service.ts

```diff
@@ -32,12 +32,15 @@ export class BamboraService {
 
   createBamboraPaymentRequest(paymentModel) {
     const amount = paymentModel.amount * 100;
-    const preTaxAmount = Math.round(
-      Number(
-        ((amount * Number(BAMBORA_TAX)) / (Number(BAMBORA_TAX) + 1)).toFixed(2),
-      ),
-    );
-    const taxAmount = BAMBORA_TAX;
+    const bamboraTax = Number(BAMBORA_TAX);
+    let preTaxAmount = amount;
+    if (bamboraTax) {
+      preTaxAmount = Math.round(
+        Number(
+          ((amount * Number(BAMBORA_TAX)) / (Number(BAMBORA_TAX) + 1)).toFixed(2),
```

Comment:
if you've converted the value to Number, like I suggested above, you can safely assume the value to be number here. It's quite cumbersome to read.
The try catch statement will also help you catch error if you bumb into any